content/guides/accessibility-for-teams/product-management.md

```diff
@@ -0,0 +1,62 @@
+---
+date: 2019-07-31 09:00:00 -0500
+title: "Accessibility for product managers"
+deck: ""
+summary: ""
+guide: accessibility-for-teams
+image: guide-dap
+primary_image: guide-dap
+layout: single
+---
+
+Product managers play a vital role in communicating accessibility requirements early in the project lifecycle, ensuring each team member knows their responsibility, and keeping the team accountable for building accessible products. Following these steps, you’ll make sure you’re not only following legal requirements, but making your product more usable for everyone.
+
+## Why you should care
+
+Any product released by the government must conform to the standards of [Section 508](https://www.section508.gov/) (which aligns with the [W3C Web Content Accessibility Guidelines (WCAG) 2.0 Level AA](https://www.w3.org/WAI/WCAG20/quickref/)), a law that ensures all web content is accessible to anybody with a disability. Not following these requirements poses significant legal risks for any government entity.
```

Comment:
@bonnieAcameron For some reason, the `Section 508` link doesn't display an external icon. Looking into it.